blueprints/ember-backstop/index.js

```diff
@@ -3,12 +3,5 @@
 module.exports = {
   description: 'Add dependencies for BackstopJS visual regression testing addon.',
 
-  normalizeEntityName() {}, // no-op since we're just adding dependencies
-
-  afterInstall() {
-    return this.addPackagesToProject([
-      {name : 'backstopjs'}
-    ]);
-  }
-
+  normalizeEntityName() {} // no-op since we're just adding dependencies
```

Comment:
yes. I tested it with https://github.com/garris/ember-backstop-tutorial repo.

Reply:
Tested it with https://github.com/garris/ember-backstop-tutorial repo.

Reply:
Hmm,  I am not sure that will work as a test because the tutorial already had backstopjs as a dependency.   So it was probably already installed at the project level.  Maybe best to use the super-rentals project and install into that.

Reply:
Per offline conversation -- @shankarsridhar checked super-rentals.  Checks out!